src/ParseServer.js

```diff
@@ -132,6 +133,7 @@ class ParseServer {
     expireInactiveSessions = true,
     verbose = false,
     revokeSessionOnPasswordReset = true,
+    schemaCacheTTL = -1, // -1 = no cache
```

Comment:
We now have NaN and -1 as special TTL values, should we put these as constants somewhere?